core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -270,7 +270,8 @@ public FormValidation doVerifySignature() throws IOException {
     /**
      * Verifies the signature in the update center data file.
      */
-    private FormValidation verifySignature(JSONObject o) throws IOException {
+    @Restricted(NoExternalUse.class)
+    public final FormValidation verifySignatureInternal(JSONObject o) throws IOException {
```

Comment:
Having to open this up indicates the handling of `platform-plugins.json` would be better off in `UpdateSite`, but to keep this change relatively simple, I decided on this approach. It can always be cleaned up later.